lightningd/gossip_control.c

```diff
@@ -145,6 +145,38 @@ const u8 *get_channel_update(struct channel *channel)
 	return channel->channel_update;
 }
 
+static void set_channel_remote_update(struct lightningd *ld,
+				      struct channel *channel,
+				      struct remote_priv_update* update TAKES)
+{
+	log_debug(ld->log, "updating channel %s with inbound settings",
+		  type_to_string(tmpctx, struct short_channel_id,
+			         channel->scid));
+	tal_free(channel->private_update);
+	channel->private_update = tal_dup(channel,
+					  struct remote_priv_update, update);
+
+	wallet_channel_save(ld->wallet, channel);
```

Comment:
Update channel def to have private_update field.
Initialize to NULL.
Populate from db.
save into db.